### ISSUE TITLE
Error setting up Email Marketing integrations - "Unknown integration"

### ISSUE THREAD

**Issue by j-greig** [verbb/formie#261]
**Description**
1) I have enabled a Campaign Monitor integration locally. When I add the API creds and refresh the connection it connects fine locally.

2) When I deploy this integration to staging via project config and attempt to refresh the connection via admin/formie/settings/email-marketing/edit/2, I get:

```An error occurred.
Unknown integration: “”```

(I was also seeing this last error when attempting the same process with a Mailchimp account)

----

**Additional info**
- Plugin version: 1.3.3
- Craft version: 3.5.16

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.4](https://github.com/verbb/formie/releases/tag/1.3.4)
